homework_4.py

```diff
@@ -0,0 +1,57 @@
+test_info_1 = "Please, write your name: "
+test_info_2 = "Write the letters you want to delete from your name?: "
+test_info_3 = "Your new name is: "
+answer_info_1 = input(f"{test_info_1}")
+answer_info_2 = input(f"{test_info_2}")
+answer_info_3 = answer_info_1.replace(answer_info_2, "")
+result_info = input(f"{test_info_3} {answer_info_3}")
```

Comment:
input() function waiting for data from user. 
For data output function print()

Reply:
fixed